public/scripts/common/lib/handlebars.mixed.js

```diff
@@ -29,10 +29,53 @@ Displays select tag with options (and optional optgroups)
   * `selected`
 
 **/
+
 Handlebars.registerHelper('select', function(context, options) {
-  var output = `<select id="${options.hash.id}" name="${options.hash.name}" class="${options.hash.class}">`;
-  output += makeOptions(context, options);
-  return output +'</select>';
+  var addOption, addOptions,
+      output = '';
+
+  addOption = function(selected, _options) {
```

Comment:
Yep.  Don't think we're using the `GENtle2-synbiota` repo any more.  I extracted this into the makeOption method:  https://github.com/Synbiota/GENtle2/commit/db7919fd67bf23f6caa6121fbb3a57665edd5df3